explainer.md

```diff
@@ -312,6 +300,67 @@ vrSession.addEventListener('ended', OnSessionEnded);
 
 If the UA needs to halt use of a session temporarily the session should be suspended instead of ended. (See previous section.)
 
+## Rendering to the Page
+
+There are a couple of scenarios in which developers may want to present content rendered with the WebVR API on the page instead of (or in addition to) a headset.
+
+### VRRenderingContext
+
+Any time WebVR content is presented on the page it's done through a `VRRenderingContext`. Like a `WebGLRenderingContext`, developers acquire a `VRRenderingContext` by calling `HTMLCanvasElement.getContext()` or `OffscreenCanvas.getContext()` with the context id of "vr". (Editor's note: Naming is hard. Please _please_ *please* suggest something better!) The returned `VRRenderingContext` is permenantly bound to the canvas.
```

Comment:
The thought had crossed my mind too. I worry that's unintuitive for magic window case, though. Also toyed with "vroutput" but it just didn't sit with me well.

Hmm... what about "vrpresent" or "vrpresentation" paired with a "VRPresentationContext"? I worry that a "Rendering" context sounds like something that you should be producing imagery with rather than a mechanism for displaying it. "Presentation" carries a bit better implications in that regard.